sdks/go/pkg/beam/testing/passert/equals_test.go

```diff
@@ -130,3 +131,69 @@ func TestEqualsList_Bad(t *testing.T) {
 		}
 	}
 }
+
+func ExampleEquals() {
+	p, s := beam.NewPipelineWithRoot()
+	col := beam.Create(s, "some", "example", "strings")
+
+	Equals(s, col, "example", "some", "strings")
+	err := ptest.Run(p)
+	fmt.Println(err == nil)
+
+	// Output: true
+}
+
+func ExampleEquals_pcollection() {
+	p, s := beam.NewPipelineWithRoot()
+	col := beam.Create(s, "some", "example", "strings")
+	exp := beam.Create(s, "example", "some", "strings")
+
+	Equals(s, col, exp)
+	err := ptest.Run(p)
+	fmt.Println(err == nil)
+
+	// Output: true
+}
+
+func ExampleEqualsList() {
+	p, s := beam.NewPipelineWithRoot()
+	col := beam.Create(s, "example", "inputs", "here")
+	list := [3]string{"here", "example", "inputs"}
+
+	EqualsList(s, col, list)
+	err := ptest.Run(p)
+	fmt.Println(err == nil)
+
+	// Output: true
+}
+
+func unwrapError(err error) error {
+	if wrapper, ok := err.(interface{ Unwrap() error}); ok {
+		err = wrapper.Unwrap()
```

Comment:
Style wise, idiomatic go tends to just return early as needed, rather than trying for a single return. 
Nothing wrong with the approach, and doesn't make a difference here but good to know.

Merging all the same though.

Reply:
Changed just to start building that habit.